src/util/key.rs

```diff
@@ -11,19 +11,58 @@
 // If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
 //
 
-//! Private key
+//! Bitcoin Keys
 //!
-//! A private key represents the secret data associated with its proposed use
+//! Structures and methods representing the public/secret data associated with
+//! its proposed use
```

Comment:
By "proposed use" what is mean there ? Being able to export it as WIF compressed pubkey ? Could be clarify :)